### PR TITLE
Solve xen login via ipmi after reboot

### PR DESCRIPTION
**Pull request by XGWang0**
Fix xen re-login issue after reboot operatiion due to no code cover that.

- Related ticket: N/A
- Needles: N/A
- Verification run: http://10.67.134.217/tests/11249

### CODE REVIEW

**Comment by XGWang0** [comment]
@alice-suse @Dawei-Pang @lansuse please help to review. Thanks

File: lib/Mitigation.pm [hunk 1]
```diff
@@ -80,9 +80,34 @@ sub reboot_and_wait {
...
+                    reset_consoles;
+                    select_console 'sol', await_console => 0;
+                }
+                send_key 'down';
+                last if check_screen 'virttest-bootmenu-xen-kernel', 5;
+            }
+            save_screenshot;
+            send_key 'ret';
+        }
+        sleep 30;    # Wait for the GRUB to disappier (there's no chance for the system to boot faster
```
Comment (Dawei-Pang) [inline lib/Mitigation.pm:102]: Please correct typo

Comment (XGWang0) [reply lib/Mitigation.pm]: > Please correct typo

Updated.

**Comment by Dawei-Pang** [comment]
LGTM

**Pull request merged**